MoreLinq/Permutations.cs

```diff
@@ -24,7 +24,6 @@ namespace MoreLinq
 
     public static partial class MoreEnumerable
     {
-        #region Nested Classes
```

Comment:
Not really. This was me cheating to make my life a little bit easier with the generator. :smirk: Trivia like this is somewhat cumbersome to deal with because of the way Roslyn does their association with parts of AST. The `#region` start gets linked with the class declaration and doesn't make it into the output because I only care about method declarations. This yields syntax errors when the generated code is compiled so I decided to take a short and drop the region (which isn't providing much value in any case). If there's a clean & simple way to deal with this kind of trivia association then this can be reverted.